### DISCUSSION TITLE
Request for Configuring a Base Path for OpenMetadata

### DISCUSSION THREAD

**Discussion by JMiller-vect** [open-metadata/OpenMetadata#17954] [Q&A]
I am currently deploying OpenMetadata in an environment where I need to serve the application under a sub-path (e.g., https://my-domain.com/service1/). However, I am encountering difficulties in configuring the application to function correctly in this context.

I have reviewed the official documentation, but I couldn't find clear information on whether it is possible to configure a base path or application context (base path or context path) for OpenMetadata.

Could you please confirm if there is a method or specific setting to define a base path for the OpenMetadata application? If so, could you provide instructions or guide me to the relevant documentation?

Thank you in advance for your assistance.

**Comment by Prajwal214** [comment]
@JMiller-vect OpenMetadata does not natively support configuring a base path or context path directly within the application. However, you can achieve this by using a reverse proxy like NGINX to serve OpenMetadata under a sub-path.

https://docs.open-metadata.org/latest/deployment/bare-metal#run-openmetadata-with-a-load-balancer

**Reply by buddzbuddy** [reply]
i've tried, but UI's packages are still referred to path like http://mydomain.con/my-sub-dir/runtime.86e1bd652e113a4d032d.js?1917559b7d64610c0a08, how to configure UI to refer proxied path http://mydomain.con/my-sub-dir/.....?

**Reply by buddzbuddy** [reply]
this is browser's console:
![image](https://github.com/user-attachments/assets/bb3522ad-a879-4ba7-bfbf-cd5b15882f20)

**Comment by jkryanchou** [comment]
I have met the same issues. while loading the static files including the javascript files and icons

**Comment by c-kon** [comment]
Was anyone able to make this work? I'm still facing the same issue with v1.11.3 even though I followed the documentation and made sure everythiing is loaded correctly, environment variables are set correctly and openmetadata.yaml is correct. It still tries to pull the assets from the domain root path instead of the specified subpath.
